### PR TITLE
add my readme

### PR DESCRIPTION
**Pull request by sudip-mondal-2002**

### CODE REVIEW

File: Readmes/Sudip Mondal/README.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
+# Sudip Mondal
```
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:1]:
```suggestion
# Sudip Mondal

```

File: Readmes/Sudip Mondal/README.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
+# Sudip Mondal                                                                       
+2nd year, B.Tech Civil Engineering                                                    
+Indian Institute of Technology, Madras, India                                       
+[E-mail](mailto:sudipmondal.2002@rediffmail.com) | [LinkedIn](https://www.linkedin.com/in/sudip-mondal-374a12201/) | [Facebook](https://www.facebook.com/sudip16022002/) | [Instagram](https://www.instagram.com/sudip._.mondal/) | [Twitter](https://twitter.com/SUDIPMO19931261) | [YouTube](https://www.youtube.com/channel/UCvfY9VkXaFNxQJO7lfDV6Vw) | [GitHub](https://github.com/sudip-mondal-2002) | [Website](https://sudipmondal.tk) | [NPM](https://www.npmjs.com/~sudipmondal) | [PyPI](https://pypi.org/user/sudip-mondal-2002/)
+## Skills
```
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:5]:
```suggestion
## Skills

```

File: Readmes/Sudip Mondal/README.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+            alt="XML" width="40" height="40" /> </a>
+</p> 
+
+## Projects
+
+<table border="0">
+ <tr>
+     <td>
+         
+### Personal Projects
```
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:116]:
```suggestion
### Personal Projects

```

File: Readmes/Sudip Mondal/README.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+* Sudip Docs: Website to write documents
+   * Learnt about sockets and real-time data sharing using this.
+   * Used MongoDB database to store the data.
+* Anime Face Generator: Generative Advisory Network
+   * This model generates a random anime face using GANs.
+   * Learnt about image processing and GANs.
+     </td>
+    <td> 
+        
+### Group projects
```
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:138]:
```suggestion
### Group projects

```

File: Readmes/Sudip Mondal/README.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+   * Learnt about integrating ML model in native android using .tflite files.
+* Fake news detector
+   * This website can detect if the news is fake or not.
+   * Learnt bout integrating python ML models to nodejs runtime.
+   * This project was a submission for pravega Hackathon, IISc.
+* Project Surya
+   * This is an app to fetch and show the data of NASA about solar radiation so people can understand the efficiency and likelihood to invest their money in solar power at a particular place.
+   * Learnt about react-native and google maps API.
+   * This was a submission for the space-apps challenge hosted by NASA.
+### Project under Professors
```
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:150]:
```suggestion
### Project under Professors

```

File: Readmes/Sudip Mondal/README.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+* Logistic & Innovation Lab, IIT Madras (prof. N.S Narayanaswamy)
+   * Logistic Innovation Lab ideates and implements efficient route algorithms for transportation.
+   * Worked in the frontend team of the project. Used React.js as the base framework.
+     </td>
+     </tr>  
+         </table>  
+## Professional Experiences
```
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:157]:
```suggestion
## Professional Experiences

```

File: Readmes/Sudip Mondal/README.md [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+* Resonect (Jun’21 - Present)
+   * Reconnect is a Nirmaan incubated startup founded by students of IIT Madras.
+   * Worked in the frontend part of this project. Used Angular and Ionic framework.
+* Mapstreak (Mar'21 - Apr'21)
+   * Mapstreak connects the restaurants with the users.
+   * Worked as a backend developer with nodejs.
+* Intellify (Jan’21 – Mar’21)
+   * Intellify is a non-government organization in the field of ed-tech.
+   * Worked as Subject Matter Expert for mathematics.
+## Position of Responsibilities
```
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:170]:
```suggestion
## Position of Responsibilities

```

File: Readmes/Sudip Mondal/README.md [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+* Core Member, WebOps Team, Sports Organizing Committee, IIT Madras (May’21 - present)
+   * Recruited and managed coordinators and maintained a steep curve for their training.
+   * Used MERN stack for building the website for Agrata, sports fest of IIT Madras.
+
+## Co-curricular Activities
```
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:175]:
```suggestion
## Co-curricular Activities

```

File: Readmes/Sudip Mondal/README.md [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+* Host a workshop for programming in Shastra Juniors. Taught school students a brief introduction to programming and the computing world.
+* Contributed to open-source projects.
+* Participated in National Cadet Corps while in school.
+
+## Scholastic Achievements
```
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:180]:
```suggestion
## Scholastic Achievements

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Awesome readme! 🚀
These are all some suggestions by the Markdown linter. 🙂

File: Readmes/Sudip Mondal/README.md [hunk 10]
```diff
@@ -0,0 +1,205 @@
+# Sudip Mondal
+
+2nd year, B.Tech Civil Engineering                                                    
+Indian Institute of Technology, Madras, India                                       
+[E-mail](mailto:sudipmondal.2002@rediffmail.com) | [LinkedIn](https://www.linkedin.com/in/sudip-mondal-374a12201/) | [Facebook](https://www.facebook.com/sudip16022002/) | [Instagram](https://www.instagram.com/sudip._.mondal/) | [Twitter](https://twitter.com/SUDIPMO19931261) | [YouTube](https://www.youtube.com/channel/UCvfY9VkXaFNxQJO7lfDV6Vw) | [GitHub](https://github.com/sudip-mondal-2002) | [Website](https://sudipmondal.tk) | [NPM](https://www.npmjs.com/~sudipmondal) | [PyPI](https://pypi.org/user/sudip-mondal-2002/)
```
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:5]:
```suggestion
[E-mail](mailto:sudipmondal.2002@rediffmail.com) | [LinkedIn](https://www.linkedin.com/in/sudip-mondal-374a12201/) | [Facebook](https://www.facebook.com/sudip16022002/) | [Instagram](https://www.instagram.com/sudip._.mondal/) | [Twitter](https://twitter.com/SUDIPMO19931261) | [YouTube](https://www.youtube.com/channel/UCvfY9VkXaFNxQJO7lfDV6Vw) | [GitHub](https://github.com/sudip-mondal-2002) | [Website](https://sudipmondal.tk) | [NPM](https://www.npmjs.com/~sudipmondal) | [PyPI](https://pypi.org/user/sudip-mondal-2002/)

```

File: Readmes/Sudip Mondal/README.md [hunk 11]
```diff
@@ -0,0 +1,206 @@
...
+   * Worked as a subject mentor and full stack developer.
+* Resonect (Jun’21 - Present)
+   * Reconnect is a Nirmaan incubated startup founded by students of IIT Madras.
+   * Worked in the frontend part of this project. Used Angular and Ionic framework.
+* Mapstreak (Mar'21 - Apr'21)
+   * Mapstreak connects the restaurants with the users.
+   * Worked as a backend developer with nodejs.
+* Intellify (Jan’21 – Mar’21)
+   * Intellify is a non-government organization in the field of ed-tech.
+   * Worked as Subject Matter Expert for mathematics.
```
Comment (Panquesito7) [inline Readmes/Sudip Mondal/README.md:176]:
```suggestion
   * Worked as Subject Matter Expert for mathematics.

```

**Review by Panquesito7 — APPROVED** [review]
LGTM! 🚀

**Comment by sudip-mondal-2002** [comment]
How to get it merged?

**Comment by Panquesito7** [comment]
> How to get it merged?

A maintainer has to merge it. I'm not a maintainer, so I can't merge it. 😅

**Review by DevrajDC — APPROVED** [review]
Looks good, merging it 🚀

**Pull request merged**